### DISCUSSION TITLE
Which setting alters how far the trailer is forward when using CP to unload a combine?

### DISCUSSION THREAD

**Discussion by coatsy35** [Courseplay/Courseplay_FS22#2675] [Q&A]
Got a number of trailers that when first unloading empty on the headboard of the trailer, which setting alters this? or what is the best way to fix it

![fsScreen_2023_08_01_21_41_57](https://github.com/Courseplay/Courseplay_FS22/assets/122838698/7710bf34-cbc5-4285-a330-e2c3e672940e)

**Comment by Tensuko** [comment]
Get in the unloader and read the hud settings....
There are offsets for both, side and forward offset.
